File: https-quera.org-problemset-10325/Solution.py
def app():
    nums = tuple(map(int, input().split()))
    r, c = nums[0], nums[1]

    a = 11 - r

    if c >= 11:
        print("Left", end=" ")
        b = 21 - c

    elif c <= 10:
        print("Right", end=" ")
        b = c

    print(a, b)

if __name__ == "__main__":
    app()

File: https-quera.org-problemset-10325/Solution.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        app();
    }


    public static void app() {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");

        int r = Integer.parseInt(input[0]);
        int c = Integer.parseInt(input[1]);

        int a = 11 - r;
        int b;

        if (c >= 11) {
            System.out.print("Left ");
            b = 21 - c;
        } else {
            System.out.print("Right ");
            b = c;
        }

        System.out.println(a + " " + b);
    }

}
